src/tree/containers/ContainerGroupTreeItem.ts

```diff
@@ -46,6 +46,8 @@ export class ContainerGroupTreeItem extends LocalGroupTreeItemBase<DockerContain
                 return getContainerStateIcon(this.group);
             case 'Image':
                 return new ThemeIcon('multiple-windows');
+            case 'Label':
```

Comment:
This is valid, but an equivalent that's also valid and a little shorter is:

```typescript
            case 'Image':
            case 'Label':
                return new ThemeIcon('multiple-windows');
```

You can keep it as is or change it--it's up to you--just wanted to make alternatives known

Reply:
Simpler, I like it.